xgi/drawing/xgi_pylab.py

```diff
@@ -195,6 +194,20 @@ def draw(
         **kwargs,
     )
 
+    # compute axis limits
+    pos_arr = np.asarray([[x, y] for n, (x, y) in pos.items()])
+
+    maxx, maxy = np.max(pos_arr, axis=0)
+    minx, miny = np.min(pos_arr, axis=0)
+    w = maxx - minx
+    h = maxy - miny
+
+    # update view after drawing
+    padx, pady = 0.05 * w, 0.05 * h
+    corners = (minx - padx, miny - pady), (maxx + padx, maxy + pady)
```

Comment:
Update on this is that I can't figure out how to do this in a robust way since the marker size is in pixel coordinates and the rest is in data coordinates. So maybe let's not deal with this yet...